lib/archiver.js

```diff
@@ -14,20 +13,33 @@ const REQUIRED_ENTRIES = [
 
 function getEntries(whitelist) {
   if (whitelist.files) {
-    const whitelistSections = _.pick(whitelist, ['main', 'files']);
-    const whitelistFromSections = _.reduce(whitelistSections, (whitelist, current) => {
-      if (current) {
-        return _.concat(whitelist, current);
-      }
-
-      return whitelist;
-    }, []);
+    // Extract (picks) the property from the whitelist
+    const { main, files } = whitelist;
+    const whitelistFromSections = [];
+
+    /*
+     * It flattens the content of both properties in the "whitelistFromSections" array,
+     * and it replaces both "_.reduce" and "_.concat" methods.
+     *
+     * If the conditional (ternary) operator is too extreme in terms of readability
+     * we can opt for the following:
+     *
+     *  if (main) {
+     *    whitelistFromSections.push(main);
+     *  }
+     *
+     *  if (files) {
+     *    whitelistFromSections.push(...files);
+     *  }
+     */
+    main ? whitelistFromSections.push(main) : null;
+    files ? whitelistFromSections.push(...files) : null;
```

Comment:
Let me know which style do you prefer for conditional here, so I can apply it and remove the comment about it.